shared/util/feature-flags.native.tsx

```diff
@@ -15,6 +15,7 @@ const ff: FeatureFlags = {
   dbCleanEnabled: false,
   fastAccountSwitch: true,
   foldersInProfileTab: false,
+  tabletSupport: __DEV__,
```

Comment:
we never do this as you'll never see it in testflight

Reply:
that's enough to get started for now. What should be done? I saw `inAdmin` in feature-flags.desktop

Reply:
just use true